zapisy/apps/theses/forms.py

```diff
@@ -33,30 +33,35 @@ class Meta:
         model = Thesis
         fields = '__all__'
 
-    title = forms.CharField(label="Tytuł pracy", max_length=MAX_THESIS_TITLE_LEN)
+    title = forms.CharField(label="Tytuł pracy", max_length=MAX_THESIS_TITLE_LEN,
+                            widget=forms.Select(attrs={'class': 'form-select'}))
```

Comment:
No, tutaj chyba to wjechało z rozpędu, tytuł nie jest _dropdownem_…
```suggestion
    title = forms.CharField(label="Tytuł pracy", max_length=MAX_THESIS_TITLE_LEN)
```

Reply:
Zmienione